docs/production-deployment/cloud/gcp-export-gcs.mdx

```diff
@@ -65,6 +63,7 @@ The following steps guide you through setting up Workflow History Export using t
 3. After inputting the necessary values, click on **Verify**. 
   You should be able to write to the sink successfully. 
   If not, please fix any errors or reach out to support for help.
+    * If you just created the GCS bucket and granted the permission for your service account, it may take some time for it to populate the permission, thus, you may need to wait for several minutes (usually 1-5 minutes) then you can click the “Verify” button to verify the connection.
```

Comment:
"several minutes (usually 1-5 minutes)" is a bit gnarly, can we just say "up to 5 minutes"

Reply:
Also, not sure those should be smart quotes around Verify.

Reply:
We bold UI, so i changed it.